packages/docusaurus-init/templates/classic/docs/doc1.md

```diff
@@ -1,203 +0,0 @@
----
```

Comment:
we can probably keep some content of this page and add it to a doc called `markdown-features.mdx`